launch/ntrip_client.launch

```diff
@@ -4,14 +4,15 @@
   <!-- Standalone example launch file for an ntrip client -->
 
   <!-- Declare arguments with default values -->
-  <arg name="host"                   default = "20.185.11.35" />
-  <arg name="port"                   default = "2101" />
-  <arg name="mountpoint"             default = "VTRI_RTCM3" />
+  <arg name="host"                   default = "IP_ADD" />
+  <arg name="port"                   default = "PORT" />
+  <arg name="mountpoint"             default = "MNT_POINT" />
   <arg name="authenticate"           default = "true" />
   <arg name="username"               default = "user" />
   <arg name="password"               default = "pass" />
   <arg name="ntrip_version"          default = "" />
   <arg name="debug"                  default = "false" />
+  <arg name="rtcm_msg_pkg"           default = "rtcm_msgs" />
```

Comment:
The default should still be `mavros_msgs`

Reply:
```suggestion
  <arg name="rtcm_msg_pkg"           default = "mavros_msgs" />
```